aws/resource_aws_elasticache_security_group_test.go

```diff
@@ -47,17 +40,12 @@ func testSweepElasticacheCacheSecurityGroups(region string) error {
 
 		for _, securityGroup := range page.CacheSecurityGroups {
 			name := aws.StringValue(securityGroup.CacheSecurityGroupName)
-			skip := true
-			for _, prefix := range prefixes {
-				if strings.HasPrefix(name, prefix) {
-					skip = false
-					break
-				}
-			}
-			if skip {
+
+			if name == "default" {
```

Comment:
Is this a known convention that is not codified? Should this be turned into a const so that it is clear that this is a special case to skip?

Reply:
The AWS Go SDK code for Elasticache doesn't provide a constant (https://docs.aws.amazon.com/sdk-for-go/api/service/elasticache/#pkg-constants), but if we want to add our own for this usage we can.